File: WebService_v2/src/main/java/Frontend.java
import javax.servlet.ServletException;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import java.io.IOException;

public class Frontend extends HttpServlet {
    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
        response.getWriter().println("hello");
    }
}

File: WebService_v2/src/main/java/servlets/Frontend.kt
package servlets

import freemarker.template.Configuration
import java.io.File
import javax.servlet.http.HttpServlet
import kotlin.Throws
import java.io.IOException
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

class Frontend : HttpServlet() {
    @Throws(IOException::class)
    public override fun doGet(request: HttpServletRequest, response: HttpServletResponse) {
        val config = Configuration(Configuration.getVersion())
        config.setDirectoryForTemplateLoading(File("templates"))
        response.writer.println(config.getTemplate("page.html"))
    }
}